### PR TITLE
Program Section Added #24

### LINKED ISSUE #24: Add a program section.
### 📚 Documentation File you'd like to improve:

- [ ] README.md
- [ ] CODE_OF_CONDUCT.md
- [ ] CONTRIBUTING.md

### 💡 Describe your ideas briefly on how to improve the documentation

Add a Program section in the Readme. and put the picture of LGM-SOC ( the logo) at the end of the Readme

---

**Note:**
* Do mention if you're working on this issue under ` LGM SOC 2021 `.
* If you want to work on any issue, you should check if it has already been assigned to anyone.

### PR DESCRIPTION
**Pull request by ishitakapoor26**
Fixes #24

## Checklist:

- [x] Have you checked there aren't other open [Pull Requests](https://github.com/Feminine-Divine/Woodog/pulls) for the same update/change?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

---

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -286,3 +286,22 @@ git push origin '<your_branch_name>'
 <img src="https://user-images.githubusercontent.com/58631762/120077716-60cded80-c0c9-11eb-983d-80dfa5862d8a.png" width="19">
 </a>
 </p>
+
+## 📌Program
```
Comment (hariramjp777) [inline README.md:290]: The program section should come before the horizontal rule and the GitHub logo.

File: README.md [hunk 2]
```diff
@@ -286,3 +286,22 @@ git push origin '<your_branch_name>'
...
+<br>
+<img src="https://camo.githubusercontent.com/d382fd0021bdf6d98f4b6904e638f9c811d2018d8325ba89106efc5a0570d498/687474703a2f2f7777772e677572707265657473616c756a612e636f6d2f77702d636f6e74656e742f75706c6f6164732f323031362f30352f616c776179732d6b6565702d6c6561726e696e672e706e67">
```
Comment (hariramjp777) [inline README.md:292]: Please remove the 'Always keep learning' photo

File: README.md (hunk 1 above)
Comment (hariramjp777) [inline README.md:290]: Change the title as Opensource programs, also a space between icon and text will look good
```
## 📌 Opensource Programs
```

**Review by hariramjp777 — CHANGES_REQUESTED** [review]
Changes requested @ishitakapoor26

**Review by hariramjp777 — APPROVED** [review]
Thanks:+1: @ishitakapoor26

**Comment by hariramjp777** [comment]
Please merge this @Aaishpra

**Review by Aaishpra — APPROVED** [review]
Thanks for working here @ishitakapoor26

**Pull request merged**